types.ml

```diff
@@ -48,3 +75,17 @@ module Change_with_hint = struct
 
   let remove_hint (t:t) = fst t
 end
+
+let or_die = function
+  | Ok x -> x
+  | Error (`Msg m) -> failwith m
+
+let sexp cmd =
+  Bos.OS.Cmd.run_out (cmd)
+  |> Bos.OS.Cmd.to_string
+  |> or_die
+  |> String.trim
+  |> (fun s ->
+      try Sexp.of_string s with
+      | Sexp.Parse_error _ as e ->
+        Fmt.pr "Error parsing '%s' output:\n" (Bos.Cmd.to_string cmd); raise e)
```

Comment:
Maybe it'd be better to log on standard error?

Reply:
Would you like to give an example ?

Reply:
I was thinking of [`Fmt.epr`](https://erratique.ch/software/fmt/doc/Fmt/index.html#val-epr) but that's really up to you :)

Reply:
I get it.